### PR TITLE
Jviereck/solo12

### PR DESCRIPTION
**Pull request by jviereck**
This adds arduino slider support to solo12.

### CODE REVIEW

File: demos/demo_arduino_slider.cpp [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        serial_reader.fill_vector(values);
+        for (int i = 0; i < 4; i++) {
+            std::cout << values[i] << " ";
+        }
+        std::cout << std::endl;
+
+        real_time_tools::Timer::sleep_sec(0.001);
+    }
+
+    // using namespace LibSerial;
```
Comment (MaximilienNaveau) [inline demos/demo_arduino_slider.cpp:47]: can we remove all this?

Comment (jviereck) [reply demos/demo_arduino_slider.cpp]: Done

**Comment by MaximilienNaveau** [comment]
Should I use the mpi_clang_format?

**Comment by jviereck** [comment]
> Should I use the mpi_clang_format?

yes please. Also, I didn't test if things compile (I don't have a working linux setup at my hand right now)...

**Pull request merged**